zebra-instructions.md

```diff
@@ -74,18 +77,72 @@ These fixes disable mining pool operator payments and miner payments: they just
     [mining]
     miner_address = 't27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v'
     ```
+
     </details>
+
 3. [Run Zebra](https://zebra.zfnd.org/user/run.html) with the `getblocktemplate-rpcs` feature:
+
     ```sh
     cargo run --release --features "getblocktemplate-rpcs"
     ```
+
 4. Wait a few hours for Zebra to sync to the testnet tip (on mainnet this takes 2-3 days)
 
-## Install and run a mining pool
+## Install `s-nomp`
+
+### Arch-specific instructions
+
+#### Install dependencies:
+
+1. Install [`redis`](https://redis.io/docs/getting-started/) and run it on the default port:
+
+    ```sh
+    sudo pacman -S redis
+    sudo systemctl start redis
+    ```
+
+2. Install and activate [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating):
+
+    ```sh
+    sudo pacman -S nvm
+    unset npm_config_prefix
+    source /usr/share/nvm/init-nvm.sh
+    ```
+
+3. Install `boost` and `libsodium` development libraries:
+
+    ```sh
+    sudo pacman -S boost libsodium
+    ```
```

Comment:
I'd like most people to be able to see instructions that work for them first. And I wonder if people will be confused by duplicate commands that do the same thing.

We could:
- put the Arch-specific instructions after the general instructions, and hide them in a `<details>` block?
- add the Arch commands to each step, after the Debian/Ubuntu commands, and maybe hide one or both in a `<details>` block (this is what we did with nvm/nodenv)

What do you think would be best here?

Reply:
Yea, thanks for the suggestion. I'll use the `<details>` tag and try to make it concise.

Reply:
I tried to refactor the instructions in the last series of commits. If that doesn't work, I can try some other approach.